domains/devbastic.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "devbastic",
+        "email": "devbastic@gmail.com"
+    },
+    "record": {
+        "CNAME": "devbastic.github.io"
```

Comment:
Please add more stuff to your website, not just a couple short lines of text.

Reply:
Ah, ok. You may want to mention that in the docs. I usually make sure it 'technically' works before I make it 'esthetically' work ;)

But will do, thanks.